webpack.config.js

```diff
@@ -30,9 +30,15 @@ module.exports = async (env, options) => {
           use: "ts-loader"
         },
         {
-          test: /\.html$/,
-          exclude: /node_modules/,
-          use: "html-loader"
+          test: /\.(html)$/,
+          exclude: "/node_modules/",
+          use: [{
+            loader: 'html-loader',
+            options: {
+              minimize: true,
```

Comment:
Shouldn't minimize only be true for prod builds, and false for dev builds? (`minimize: !dev`)